examples/shebang.md

```diff
@@ -0,0 +1,57 @@
+# Making scripts executable with #!Shebang
+
+## Concepts
+
+- [Deno.env] provides the environment variables.
+- [env] runs a program in a modified environment.
+
+## Overview
+
+Making Deno scripts executable can be useful if you want to make, for example,
+small tools.
+
+## Example
+
+In this program we give the context permission to access the environment
+variables and print the Deno installation path.
+
+```typescript
+#!/usr/bin/env -S deno run --allow-env
+
+/**
+ *  Shebang.ts
+ */
+
+const deno_path = Deno.env.get("DENO_INSTALL");
```

Comment:
We usually don't use snake_case for variable names. Can you change it to camelCase?

Reply:
Sure